components/ApplicationsList/ApplicationsList.jsx

```diff
@@ -179,30 +183,41 @@ const ApplicationsList = ({
         </a>
       </p>
       <p>
-        <a
-          href="#"
+        <p>
+          {!canDownloadCsvs &&
+            `These downloads are disabled as you are not part of the '${csvDownloadGroup}' user group`}
+        </p>
+        <button
+          className="govuk-button govuk-button--secondary govuk-!-margin-right-1"
+          data-module="govuk-button"
           onClick={() =>
             handleCsvDownload({ grant_type: 'lrsg_closed_businesses' })
           }
+          disabled={canDownloadCsvs ? '' : 'disabled'}
         >
           Export LSRG (Closed) Panel Approved Payments
-        </a>
-      </p>
-      <p>
-        <a
-          href="#"
+        </button>
+        <br />
+
+        <button
+          className="govuk-button govuk-button--secondary"
+          data-module="govuk-button"
           onClick={() => handleCsvDownload({ grant_type: 'lrsg_sector' })}
+          disabled={canDownloadCsvs ? '' : 'disabled'}
         >
           Export LSRG (Sector) Panel Approved Payments
-        </a>
-      </p>
-      <p>
-        <a
-          href="#"
+        </button>
+        <br />
+
+        <button
+          className="govuk-button govuk-button--secondary"
+          data-module="govuk-button"
           onClick={() => handleCsvDownload({ grant_type: 'lrsg_open' })}
+          disabled={canDownloadCsvs ? '' : 'disabled'}
```

Comment:
Might be worth checking this in browsers - if the disabled attribute is rendered by Next (with no value or with `disabled=''`), buttons always be disabled (in some browsers if I remember rightly).
This might be a way round that: https://stackoverflow.com/questions/29103096/dynamic-attribute-in-reactjs/29103727

Reply:
Good spot, apparently that's Chrome behaviour